<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaggingWidget</class>
 <widget class="QWidget" name="TaggingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>824</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="VectorialView" name="vectorialView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>420</width>
       <height>370</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16637</width>
       <height>16637</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="backgroundBrush">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>136</red>
        <green>138</green>
        <blue>133</blue>
       </color>
      </brush>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="newAntButton">
       <property name="text">
        <string>New Ant From Tag</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addIdentButton">
       <property name="text">
        <string>Add Identification to Ant</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="deletePoseButton">
       <property name="text">
        <string>Delete Pose Estimation</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="tagBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Tag Selection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="familyLabel">
          <property name="text">
           <string>Family:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="TagFamilyComboBox" name="familySelector"/>
        </item>
        <item>
         <widget class="QLabel" name="thresholdLabel">
          <property name="text">
           <string>B/W Threshold:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="thresholdBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>254</number>
          </property>
          <property name="value">
           <number>40</number>
          </property>
          <property name="displayIntegerBase">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="unusedTagLabel">
          <property name="text">
           <string>Unused Tag(s):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="unusedTagDisplay">
          <property name="text">
           <string>N.A.</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="missingTagLabel">
          <property name="text">
           <string>Missing Tag(s):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="missingTagDisplay">
          <property name="text">
           <string>N.A.</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeView" name="treeView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QProgressBar" name="tagCloseUpLoadingProgress">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
     <property name="format">
      <string>%v / %m</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="AntIdentificationWidget" name="selectedAntIdentification" native="true">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VectorialView</class>
   <extends>QGraphicsView</extends>
   <header location="global">fort-studio/widget/vectorgraphics/VectorialView.hpp</header>
  </customwidget>
  <customwidget>
   <class>TagFamilyComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">fort-studio/widget/base/TagFamilyComboBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>AntIdentificationWidget</class>
   <extends>QWidget</extends>
   <header location="global">fort-studio/widget/AntIdentificationWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
